src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
```

Comment:
A great implementation following the ducks pattern 👍🏽

Reply:
Thanks you :+1: